### ISSUE TITLE
The following INFO field appears more than once

### ISSUE THREAD

**Issue by tedtoal** [knausb/vcfR#138]
In vcfR 1.8.0 I get many warnings such as:

```
The following INFO key occurred more than once: ##GATKCommandLine.CombineVariants=<ID=CombineVariants

The following FORMAT key occurred more than once: AD
```

The warnings include both INFO and FORMAT fields.

One problem is that the first example above is not an INFO key.

Another problem is that the INFO and FORMAT keys that actually do exist and are warned about DO NOT appear more than once in the VCF file.

**Comment by knausb** [comment]
Hi @tedtoal, 
Without an example that I can reproduce on my machine there will be little I can do on this. I've place some suggestions on reporting and issue with the [vcfR documentation](https://knausb.github.io/vcfR_documentation/reporting_issue.html). Note the bit about human data which probably should not be shared. Thanks!
Brian

**Comment by tedtoal** [comment]
This problem is no longer happening with me, even though I'm still using vcfR V1.8.0.  It must have been caused somehow by the fact that I was using a version of R that could not install vcfR via install.packages() because CXX11 was not defined because of an error in the R ./configure script that interacted with the Conda defining of CXXFLAGS.  I instead installed vcfR via conda itself, and it installed successfully, and I assumed that was because it had already been compiled and didn't need to be compiled with an R that lacked CXX11.  But the conda-installed vcfR is the one that produced this error, and when I was able to rebuild R to have CXX11, I was able to install vcfR with install.packages(), and then I no longer got the error.
